### PR TITLE
Rename us-benefits-policy.md to README.md

### PR DESCRIPTION
**Pull request by ContentBrewmaster**
Updating file name to handle new editing tool.

----
:books: Documentation preview :books:: https://civicactions-handbook--1152.org.readthedocs.build/en/1152/

### CODE REVIEW

**Review by dmundra — APPROVED** [review]
Only a filename change! Thank you @ContentBrewmaster

File: .config/mkdocs.yml [hunk 1]
```diff
@@ -27,11 +27,13 @@ nav:
       - employee-benefits/employee-referral-bonus.md
       - employee-benefits/on-call-stipend.md
   - Company policies:
+      - company-policies/README.md
       - company-policies/expenses.md
       - company-policies/travel-time-tracking-and-expenses.md
       - company-policies/timesheets.md
       - company-policies/security.md
       - Employment policies (US employees):
+          - Employment: company-policies/README.md
```
Comment (dmundra) [inline .config/mkdocs.yml]: @ContentBrewmaster looks like the company-policies/README.md change is not in this PR. Should this be in another PR?

Comment (ContentBrewmaster) [reply .config/mkdocs.yml]: @dmundra I adjusted the file name.

File: .config/mkdocs.yml [hunk 2]
```diff
@@ -27,12 +27,14 @@ nav:
...
       - employee-benefits/on-call-stipend.md
   - Company policies:
+      - company-policies/employment/us/README.md
       - company-policies/expenses.md
       - company-policies/travel-time-tracking-and-expenses.md
       - company-policies/timesheets.md
       - company-policies/security.md
       - Employment policies (US employees):
-          - company-policies/employment/us/employment.md
+          - Employment: company-policies/README.md
```
Comment (dmundra) [inline .config/mkdocs.yml]: I think this link should be `Employment: company-policies/employment/us/README.md`? Right now it goes to a 404 when you try it here https://civicactions-handbook--1152.org.readthedocs.build/en/1152/company-policies/employment/us/

**Review by dmundra — CHANGES_REQUESTED** [review]
@ContentBrewmaster thanks for fixing the menu. I found one small issue.

File: .config/mkdocs.yml (hunk 2 above)
Comment (ContentBrewmaster) [reply .config/mkdocs.yml]: @dmundra this should be ready to go?

Comment (dmundra) [reply .config/mkdocs.yml]: Maybe not. I am not seeing the 'Employment' sub menu item in the sidebar. Maybe that has to go below the README? Like this:

```
          - company-policies/employment/us/README.md
          - Employment: company-policies/employment/us/README.md
```
